src/app/app-routing.module.ts

```diff
@@ -5,7 +5,7 @@ import { Routes, RouterModule } from '@angular/router';
 const routes: Routes = [];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
```

Comment:
Better to use corrected instead of the default option https://angular.io/api/router/ExtraOptions#relativeLinkResolution

Reply:
@sajeetharan I changed it to this
```
imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'corrected' })],
```